crates/matrix-sdk-base/Cargo.toml

```diff
@@ -18,7 +18,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 [features]
 default = []
 e2e-encryption = ["dep:matrix-sdk-crypto"]
-js = ["matrix-sdk-common/js", "matrix-sdk-crypto?/js", "ruma/js"]
+js = ["matrix-sdk-common/js", "matrix-sdk-crypto?/js", "ruma/js", "matrix-sdk-store-encryption/js"]
```

Comment:
This seems unrelated, can you move it into its own PR or at least into a separate commit?